content/docs/command-reference/repro.md

```diff
@@ -262,31 +263,40 @@ The answer to universe is 42
 - The Hitchhiker's Guide to the  Galaxy
 ```
 
-Now, using the `--downstream` option results in the following output:
+Let's say we want to print the filename also in the description and so we update
+the `process.py` as:
+
+```python
+print(f'Number of lines in {sys.argv[1]}:')
+print(num_lines)
+```
+
+Now, using the `--downstream` option with `dvc repro` results in the execution
+only of stages after the target (`count`):
 
 ```dvc
-$ dvc repro --downstream
-Data and pipelines are up to date.
+$ dvc repro --downstream count
+Running stage 'count' with command:
+        python process.py numbers.txt > count.txt
+Updating lock file 'dvc.lock'
 ```
 
-The reason being that the `text.txt` file is not a dependency in the last stage
-of the pipeline, used as the default target by `dvc repro`. `text.txt` is a
-dependency of the `filter` stage, which happens earlier (shown in the figure
-below), so it's skipped given the `--downstream` option.
+The change in `text.txt` is ignored because that file is a dependency in the
+`filter` stage, which did not get updated in the above command. This is because
+`filter` happens before `count` in the pipeline (shown below).
```

Comment:
```suggestion
`filter` stage, which wasn't executed by the `dvc repro` above. This is because
`filter` happens before the target (`count`) in the pipeline, as shown below:
```